server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationPlugin.java

```diff
@@ -3,29 +3,32 @@
 import java.util.HashSet;
 import java.util.Set;
 
-import org.opentosca.toscana.core.plugin.AbstractPlugin;
 import org.opentosca.toscana.core.transformation.TransformationContext;
 import org.opentosca.toscana.core.transformation.platform.Platform;
 import org.opentosca.toscana.core.transformation.properties.Property;
+import org.opentosca.toscana.plugins.lifecycle.LifecycleAwarePlugin;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
 @Component
-public class CloudFormationPlugin extends AbstractPlugin {
+public class CloudFormationPlugin extends LifecycleAwarePlugin<CloudFormationLifecycle> {
+    private final static Logger logger = LoggerFactory.getLogger(CloudFormationPlugin.class);
```

Comment:
This logger seems unecessary, because you currently have nothing to log.

but: Keep it in. didn't whine about it in the other pull requests.